README.md

```diff
@@ -2,10 +2,15 @@
 
 ![alt text](https://github.com/WSU-4110/Live-Sports-Display/blob/main/Logo/MMM_Logo.png)

```

Comment:
I will have to change this logo to the new one as it’s still listed as one with text.